From 5804e71a85b45f9801778f5c0d739a3e2574bba2 Mon Sep 17 00:00:00 2001
From: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Date: Tue, 16 Aug 2016 14:48:44 +0530
Subject: [PATCH 743/793] nand: arasan: balance the select and delect

balance the select and de-select.
this is to aid the power gating at deselect.

Acked-by: Punnaiah Choudary Kalluri<punnaia@xilinx.com>
Signed-off-by: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/mtd/nand/arasan_nfc.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/mtd/nand/arasan_nfc.c b/drivers/mtd/nand/arasan_nfc.c
index a6f7cd3..c781db9 100644
--- a/drivers/mtd/nand/arasan_nfc.c
+++ b/drivers/mtd/nand/arasan_nfc.c
@@ -854,6 +854,7 @@ static int anfc_init_timing_mode(struct anfc *nfc)
 		err = chip->onfi_set_features(mtd, chip,
 					ONFI_FEATURE_ADDR_TIMING_MODE,
 					(uint8_t *)&feature[0]);
+		chip->select_chip(mtd, -1);
 		if (err)
 			return err;
 	}
-- 
1.7.5.4

